assets/js/index.js

```diff
@@ -0,0 +1,333 @@
+// global declarations
+const questions = [
+  {
+    question: "Which of these is not used to declare a variable in JavaScript?",
+    answers: ["Const", "Let", "Var", "Dec"],
+    correctAnswer: "Dec",
+  },
+  {
+    question: "What is the name for the symbols ( ) ?",
+    answers: [
+      "Curly Brackets",
+      "Angular Brackets",
+      "Parentheses",
+      "Open Brackets",
+    ],
+    correctAnswer: "Parentheses",
+  },
+  {
+    question:
+      "What attribute should be used to link a JavaScript file to an HTML file?",
+    answers: ["src", "href", "link", "rel"],
+    correctAnswer: "src",
+  },
+  {
+    question: "What symbol is used to reference an ID in CSS?",
+    answers: [".", "#", "/", "="],
+    correctAnswer: "#",
+  },
+  {
+    question: "Which of these is not a type of loop statement in JavaScript?",
+    answers: ["For", "While", "Rotate", "Continue"],
+    correctAnswer: "Rotate",
+  },
+];
+
+let questionIndex = 0;
+
+let timerValue = 12 * questions.length;
+
+let answerStatus = "not selected";
+
+let quizComplete = false;
+
+const startBtn = document.getElementById("start-button");
+
+const removeStart = document.getElementById("start-content");
+
+const mainContent = document.getElementById("main-content");
+
+// initialise local storage
+const onLoad = () => {
+  // check if highscores exists in LS
+  const highScores = readFromLocalStorage();
+  // if false then set highscores to empty array in LS
+  if (!highScores) {
+    localStorage.setItem("highscores", JSON.stringify([]));
+  }
+};
+
+// getScores from local storage
+const readFromLocalStorage = () => {
+  // get from LS by key
+  const getScores = localStorage.getItem("highscores");
+  // parse LS data
+  const parsedData = JSON.parse(getScores);
+  return parsedData;
+};
+
+const startTimer = () => {
+  // declare function to execute every 1 sec
+  const countdown = () => {
+    //     decrement timer value
+    timerValue -= 1;
+
+    // timer.textContent = timerValue;
+    // if quizComplete is true then stop timer
+    if (quizComplete) {
+      clearInterval(timerId);
+    } else {
+      const updateTimer = document.getElementById("timer-span");
+      updateTimer.textContent = timerValue;
+    }
+
+    // check if timer reaches 0
+    if (timerValue === 0) {
+      // if true render game over
+      clearInterval(timerId);
+      document.getElementById("question-section").remove();
+      renderRecordScoreSection();
+    }
+  };
+
+  // setInterval of 1000ms (1s)
+  const timerId = setInterval(countdown, 1000);
+};
+
+const renderTimerSection = () => {
+  // use DOM tree as guide to build in JS
+  const timerSection = document.createElement("section");
+  timerSection.setAttribute("id", "timer-section");
+
+  const timeRemaining = document.createElement("div");
+  timeRemaining.textContent = "Time Remaining: ";
+
+  const timerSpan = document.createElement("span");
+  //   TODO - check if better way to set multiple attributes
```

Comment:
I don't actually think there is unfortunately! But now you've seen jQuery, you know there's a better way to do all of this!!